.github/workflows/lint.yml

```diff
@@ -1,19 +1,42 @@
-name: Lint
+name: golangci-lint
 on:
   push:
-    branches:
-      - master
   pull_request:
-
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
 jobs:
   golangci:
-    # We need to run the linter on the same image we use for building, since it
-    # needs the C libraries installed for the dependencies to typecheck.
+    name: lint
     runs-on: ubuntu-latest
-    container: tendermintdev/docker-tm-db-testing
+    container: ghcr.io/faddat/cosmos
```

Comment:
ditto

Reply:
reasoning: when we bump rocksdb, we also need to use a different docker image. The alternative is to upgrade the current docker image.